lib/manageiq/providers/ovirt/legacy/event_monitor.rb

```diff
@@ -0,0 +1,50 @@
+require 'ovirt'
```

Comment:
The only reason is that these classes are used by the those test scripts that I don't know where to place. Those are in the `manageiq-gems-pending` repo. Not sure if they can be used from that repo if they aren't in the `lib` directory of this gem. Once we decide what we do with those test scripts, an once I know how to test them, I can move the files to `app/models` no objection to that.